pipe1-transfer-request-finisher.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+export COMPONENT_NAME=${COMPONENT_NAME:="$(hostname)-pipe1-transfer-request-finisher"}
+export DEST_SITE=${DEST_SITE:="NERSC"}
+export HEARTBEAT_PATCH_RETRIES=${HEARTBEAT_PATCH_RETRIES:="3"}
+export HEARTBEAT_PATCH_TIMEOUT_SECONDS=${HEARTBEAT_PATCH_TIMEOUT_SECONDS:="5"}
+export HEARTBEAT_SLEEP_DURATION_SECONDS=${HEARTBEAT_SLEEP_DURATION_SECONDS:="30"}
+export INPUT_STATUS=${INPUT_STATUS:="deleted"}
+export LTA_REST_TOKEN=${LTA_REST_TOKEN:="$(<service-token)"}
+export LTA_REST_URL=${LTA_REST_URL:="https://lta.icecube.aq:443"}
+export OUTPUT_STATUS=${OUTPUT_STATUS:="finished"}
+export RUN_ONCE_AND_DIE=${RUN_ONCE_AND_DIE:="False"}
+export SOURCE_SITE=${SOURCE_SITE:="WIPAC"}
```

Comment:
Looks like `DEST_SITE` and `SOURCE_SITE` weren't swapped in this one.
Change `DEST_SITE` to `WIPAC` and `SOURCE_SITE` to `NERSC` and it should be good.

Reply:
Good catch